src/bundle/test-transformer.ts

```diff
@@ -2,10 +2,14 @@ import * as nodePath from "node:path";
 import * as babel from "@babel/core";
 import { addHook } from "pirates";
 import { TRANSFORM_EXTENSIONS, JS_EXTENSION_RE } from "./constants";
+import { requireModuleSync } from "../utils/module";
 
 import type { NodePath, PluginObj, TransformOptions } from "@babel/core";
 import type { ImportDeclaration } from "@babel/types";
 
+const STYLE_EXTESTION_RE = /\.(css|less|scss)$/;
+const IGNORE_IMPORT_ERRORS = ["Unexpected token"];
```

Comment:
When style files imported in node js environment I get an error like: `Unexpected token .` (when first line in css is `.some-class {...}` or `Unexpected token {` (when first line is `body {...}`)